src/content/structured/components/cards/guidance.mdx

```diff
@@ -167,7 +167,7 @@ Include an image either at the top or in the middle of the card. Don't place ima
 
 ### 3. Content area
 
-Add a descriptive message within the card content section. Interactive and static cards can include a “See more” link to expand hidden content.
+Add a descriptive message within the card content section. Interactive and static cards can include a 'See more' link to expand hidden content.
```

Comment:
There are inconsistencies withe 'See more' and 'see more'. I would tend towards the 'see more' so that the text is sentence case.